packages/autorest.python/autorest/codegen/models/operation_group.py

```diff
@@ -70,18 +70,20 @@ def imports(self, async_mode: bool) -> FileImport:
         file_import = FileImport()
 
         relative_path = "..." if async_mode else ".."
+        # for multiapi
```

Comment:
how is this for multiapi?

Reply:
it was existing comment. Rearranged this part to make the diff clean